.github/workflows/linux-ci.yml

```diff
@@ -0,0 +1,28 @@
+name: Linux-CPU
+on: [push, pull_request]

```

Comment:
can we specify the main branch for the push/pull_request events?
https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#onpushpull_requestbranchestags

Reply:
Like
```yaml
on:
  push:
    # Sequence of patterns matched against refs/heads
    branches:    
      # Push events on main branch
      - main
  pull_request:
    # Sequence of patterns matched against refs/heads
    branches:    
      # Push events on main branch
      - main
```

Reply:
yeah, that looks good

Reply:
Updated. Please help me check.